packages/wrangler/src/pages/errors.ts

```diff
@@ -9,11 +9,13 @@ import { RoutesValidationError } from "./functions/routes-validation";
 /**
  * Error codes returned by requests to Pages APIs
  */
+/* eslint-disable no-unused-vars */
```

Comment:
If this is exported, why is it breaking no-unused-vars?

Reply:
Because I was using the wrong eslint rule 😅 

(I need to use the typescript one and not the standard js one)

Thanks for making me notice that! 🙏